# ***Statistic Basic Assignment 2 October 2024 by Piyush Gaur***


#### 1. Explain the different types of data (qualitative and quantitative) and provide examples of each. Discuss nominal, ordinal, interval, and ratio scales.

#### **Types of Data: Qualitative and Quantitative**

- **Qualitative Data**: Also known as categorical data, this type of data describes qualities or characteristics. It's non-numeric and often involves labels or categories.
  - **Examples**: Colors (red, blue), Gender (male, female), Nationality (Indian, American)
  - **Scales**:
    - **Nominal Scale**: Data is categorized without any order. Example: Eye color (blue, brown, green).
    - **Ordinal Scale**: Data is categorized but with a meaningful order. Example: Customer satisfaction (satisfied, neutral, dissatisfied).
  
- **Quantitative Data**: This data type represents numerical values and can be measured.
  - **Examples**: Height (in cm), Weight (in kg), Test scores
  - **Scales**:
    - **Interval Scale**: Data with meaningful intervals between values but no true zero. Example: Temperature in Celsius.
    - **Ratio Scale**: Data with meaningful intervals and a true zero, allowing for comparisons of magnitude. Example: Height, Weight.

---



#### 2. What are the measures of central tendency, and when should you use each? Discuss the mean, median, and mode with examples and situations where each is appropriate.


#### **Measures of Central Tendency: Mean, Median, Mode**

- **Mean**: The average of a dataset. It’s useful when the data is evenly distributed without extreme outliers.
  - **Example**: For the numbers 2, 3, 5, 7, 11, the mean is $$\frac{2+3+5+7+11}{5} = 5.6$$
  - **When to use**: When you want an overall average and the data is symmetric.

- **Median**: The middle value in an ordered dataset. It’s robust to outliers and skewed data.
  - **Example**: In the dataset 1, 3, 7, the median is 3.
  - **When to use**: When the data is skewed or has outliers.

- **Mode**: The value that appears most frequently in a dataset.
  - **Example**: In the dataset 2, 3, 3, 5, 7, the mode is 3.
  - **When to use**: When the most common item is important (e.g., in a survey about the most preferred product).

---



#### 3. Explain the concept of dispersion. How do variance and standard deviation measure the spread of data?


#### **Concept of Dispersion: Variance and Standard Deviation**

- **Dispersion** refers to how spread out the data points are in a dataset. It shows the variability within the data.
  
- **Variance** measures the average of the squared differences from the mean.
  - **Formula**: $$\text{Variance} = \frac{\sum (x_i-\mu)^2}{N}$$, where $$x_i$$ are the data points, and $$\mu $$ is the mean.

- **Standard Deviation** is the square root of the variance, providing a measure of spread in the same units as the original data.
  - **Formula**: $$\text{Standard Deviation} = \sqrt{\text{Variance}}$$

  Both metrics tell us how far the data points deviate from the mean. A higher standard deviation means the data is more spread out.

---



#### 4. What is a box plot, and what can it tell you about the distribution of data?

#### **Box Plot (Box-and-Whisker Plot)**

A box plot visualizes the distribution of a dataset based on five summary statistics: minimum, first quartile (Q1), median (Q2), third quartile (Q3), and maximum. It also shows potential outliers.

- **What it tells you**:
  - Spread and skewness of the data.
  - Presence of outliers.
  - Comparison of different data distributions.

---



#### 5. Discuss the role of random sampling in making inferences about populations.

#### **Role of Random Sampling in Inferences**

Random sampling ensures that every member of the population has an equal chance of being selected. This allows for unbiased estimates of population parameters and helps generalize findings from the sample to the population.

- **Importance**: It reduces sampling bias and increases the reliability of statistical inferences.

---


#### 6. Explain the concept of skewness and its types. How does skewness affect the interpretation of data?



#### **Skewness and Its Types**

- **Skewness** measures the asymmetry of the distribution of values:
  - **Positive Skew (Right-skewed)**: Tail is on the right; mean > median.
  - **Negative Skew (Left-skewed)**: Tail is on the left; mean < median.

  Skewness affects the interpretation of the central tendency and spread, especially for the mean.

---


#### 7. What is the interquartile range (IQR), and how is it used to detect outliers?



#### **Interquartile Range (IQR) and Outlier Detection**

- **IQR** is the difference between the third quartile (Q3) and the first quartile (Q1). It measures the spread of the middle 50% of the data.
  - **Formula**: $$\text{IQR} = Q3 - Q1$$

- **Outliers**: Data points outside 1.5 times the IQR above Q3 or below Q1 are considered outliers.

---



#### 8. Discuss the conditions under which the binomial distribution is used.


#### **Binomial Distribution**

The **binomial distribution** is used when:
- There are a fixed number of trials.
- Each trial has two possible outcomes (success or failure).
- The probability of success is constant across trials.
- Trials are independent.

**Example**: Tossing a coin 10 times and counting the number of heads.

---


#### 9. Explain the properties of the normal distribution and the empirical rule (68-95-99.7 rule).


#### **Normal Distribution and the Empirical Rule**

- **Normal Distribution** is symmetric and bell-shaped, where most data points cluster around the mean.
  - **Properties**: Mean = Median = Mode.

- **Empirical Rule (68-95-99.7 Rule)**: 
  - 68% of data falls within 1 standard deviation.
  - 95% of data falls within 2 standard deviations.
  - 99.7% of data falls within 3 standard deviations.

---


#### 10. Provide a real-life example of a Poisson process and calculate the probability for a specific event.




#### **Poisson Process Example**

A **Poisson process** models the occurrence of rare events over a fixed interval of time or space.

**Example**: Suppose on average 5 customers enter a store per hour. The probability of exactly 3 customers entering in the next hour can be calculated using the Poisson formula:

$$
P(X=k) = \frac{e^{-\lambda} \lambda^k}{k!}
$$
Where $$\lambda = 5$$, $$k = 3$$, and $$e$$ is approximately 2.718.


---

#### 11. Explain what a random variable is and differentiate between discrete and continuous random variables.


#### **Random Variables: Discrete and Continuous**

- **Random Variable**: A variable that takes on different values based on the outcome of a random event.
  - **Discrete Random Variables**: Take on countable values (e.g., number of heads in coin tosses).
  - **Continuous Random Variables**: Take on any value in a continuous range (e.g., height, weight).

---


#### 12. Provide an example dataset, calculate both covariance and correlation, and interpret the results.


#### **Covariance and Correlation Example**

- **Covariance** measures the directional relationship between two variables. A positive covariance means both variables move in the same direction and negative covariance is both variable move in opposite direction.
  - **Formula**: $$\text{Cov}(X, Y) = \frac{\sum (X_i - \bar{X})(Y_i - \bar{Y})}{N}$$

- **Correlation** standardizes covariance, giving a value between -1 and 1.
  - **Formula**: $$\text{Corr}(X, Y) = \frac{\text{Cov}(X, Y)}{\sigma_X \sigma_Y}$$

A positive correlation (near 1) means a strong positive relationship, and a negative one (near -1) means a strong negative relationship.



In [3]:
import numpy as np
# Example dataset
X = np.array([1, 2, 3, 4, 5])
Y = np.array([2, 4, 5, 4, 5])

# Mean of X and Y
mean_X = np.mean(X)
mean_Y = np.mean(Y)

# Covariance calculation
cov_XY = np.sum((X - mean_X) * (Y - mean_Y)) / (len(X) - 1)

# Correlation calculation
std_X = np.std(X, ddof=1)  # standard deviation of X
std_Y = np.std(Y, ddof=1)  # standard deviation of Y
correlation_XY = cov_XY / (std_X * std_Y)

cov_XY, correlation_XY


(1.5, 0.7745966692414834)

#### Results:
- **Covariance (X, Y):** 1.5
- **Correlation (X, Y):** 0.7746

#### Interpretation:
- **Covariance:** The positive covariance of 1.5 indicates that as \( X \) increases, \( Y \) tends to increase as well. However, covariance alone does not give a clear sense of the strength of the relationship.
  
- **Correlation:** The correlation of 0.7746 suggests a strong positive linear relationship between \( X \) and \( Y \). The correlation value lies between -1 and 1, where values closer to 1 indicate a strong positive relationship.

In this case, a correlation of 0.7746 shows that \( X \) and \( Y \) are moderately to strongly positively related.